conf/workloads/ml/lr.conf

```diff
@@ -0,0 +1,44 @@
+hibench.lr.tiny.examples                    	100
+hibench.lr.tiny.features                    	1000
+hibench.lr.tiny.frac_categorical_features   	0
+hibench.lr.tiny.frac_binary_features        	0
+hibench.lr.tiny.maxresultsize			"1g"
+hibench.lr.small.examples                   	1000
+hibench.lr.small.features                   	10000
+hibench.lr.small.frac_categorical_features  	0
+hibench.lr.small.frac_binary_features       	0
+hibench.lr.small.maxresultsize                  "1g"
+hibench.lr.large.examples                   	10000
+hibench.lr.large.features                   	100000
+hibench.lr.large.frac_categorical_features  	0
+hibench.lr.large.frac_binary_features       	0
+hibench.lr.large.maxresultsize                  "1g"
+hibench.lr.huge.examples                    	10000
+hibench.lr.huge.features                    	300000
+hibench.lr.huge.frac_categorical_features   	0
+hibench.lr.huge.frac_binary_features        	0
+hibench.lr.huge.maxresultsize                   "3g"
+hibench.lr.gigantic.examples                    10000
+hibench.lr.gigantic.features                    500000
+hibench.lr.gigantic.frac_categorical_features   0
+hibench.lr.gigantic.frac_binary_features        0
+hibench.lr.gigantic.maxresultsize               "5g"
+hibench.lr.bigdata.examples                    	10000
+hibench.lr.bigdata.features                    	700000
+hibench.lr.bigdata.frac_categorical_features   	0
+hibench.lr.bigdata.frac_binary_features        	0
```

Comment:
`frac_categorical_features` and `frac_binary_features` are always 0 for all scales?